src/com/lilithsthrone/game/character/persona/Name.java

```diff
@@ -665,6 +732,11 @@ public static String getSurname(GameCharacter gc) {
 	
 	public static NameTriplet getRandomTriplet(Race r) {
 		NameTriplet name = Util.randomItemFrom(human);
+		// occasionally throw some "prostitute" names in there - about 10% of the time
+		if(Util.random.nextInt(9)>8)
```

Comment:
The `nextInt(int bound)` method treats the specified value as exclusive, meaning that in this instance, the method will always return false. (As the returned values are 0-8, inclusive.)

I personally prefer to use Math.random() for percentage-based random chances, as I find it clearer to read. As such, I'd change this line from:
`if(Util.random.nextInt(9)>8)`
to
`if(Math.random()<0.1)`